src/common/transformations/src/transformations/common_optimizations/pad_fusion.cpp

```diff
@@ -96,14 +97,14 @@ pass::PadFusionAvgPool::PadFusionAvgPool() {
     auto pads_end_pattern = pattern::wrap_type<opset5::Constant>();
     auto pad_value_pattern = pattern::any_input();
     auto pad_node_pattern =
-        pattern::wrap_type<opset5::Pad>({data_pattern, pads_begin_pattern, pads_end_pattern, pad_value_pattern},
-                                        pattern::consumers_count(1));
+        pattern::wrap_type<op::util::PadBase>({data_pattern, pads_begin_pattern, pads_end_pattern, pad_value_pattern},
```

Comment:
I hope the check on line 58 should already detect the negative pad values

Reply:
Thank you. I noticed that before PR creation